### DISCUSSION TITLE
幸亏我没钱买房，哈哈哈

### DISCUSSION THREAD

**Discussion by daixiangzi** [WeNeedHome/SummaryOfLoanSuspension#114] [General]
。。。。

**Comment by webjohnjiang** [comment]
那我们也要为受害者鸣不平。因为不知道什么时候，糟心事就落到自己头上了。

**Reply by AntEffect** [reply]
除非推翻官僚体制，不然铁拳早晚砸到自己

**Reply by smartleek** [reply]
是这样的，铁拳飞舞，谁也无法保证自己不会永远不会被击中

**Comment by guyanyouyou** [comment]
总有一个坑为你量身定做

**Comment by BobbyYang9** [comment]
已经润了，国内资产除了一辆车十万块钱左右其他没有了，15年前就怕砸到自己头上，所以早就做好准备了

**Reply by Ace-1997** [reply]
兄弟怎么润的  去哪里了

**Reply by BobbyYang9** [reply]
> 

13年就计划了，润到了希腊圣托里尼海边，就买了套300万人民币的洋房吧，就润了

**Reply by BobbyYang9** [reply]
> 兄弟怎么润的 去哪里了

钱的话，一是通过信托，二是通过购买美元带出去了一部分，还有换了金条

**Reply by O-kai** [reply]
> > 
> 
> 13年就计划了，润到了希腊圣托里尼海边，就买了套300万人民币的洋房吧，就润了

老哥有点强啊！这是ow的伊利奥斯地图原型吗？有点羡慕啊！

**Reply by BobbyYang9** [reply]
> 

是的，蓝白那种坡房，但是我住山腰上，靠边的买不起，毕竟不是贪官

**Reply by qq912276337** [reply]
你给我回来！！！

**Reply by BobbyYang9** [reply]
> 你给我回来！！！

我劝你别润，我润你别劝 👍 P

**Comment by SlighterR** [comment]
去年就在观望买房，如今观望一年了，越来越不敢买了

**Reply by daixiangzi** [reply]
> 去年就在观望买房，如今观望一年了，越来越不敢买了

现房就可以买

**Reply by eren-amundsen** [reply]
> 

主要是看房价后面会不会跌

**Reply by yidardd** [reply]
刚需也不怕跌了,只不要跌的太狠了

**Reply by eren-amundsen** [reply]
> 

怕的就是跌的太狠。。所以要观望

**Reply by Ndoujiang** [reply]
19年想买 ，已经在看了， 后来一直在观望价钱 ，真是越观望越不敢买，我以为去年到底了，结果今年更低了

**Reply by eren-amundsen** [reply]
> 19年想买 ，已经在看了， 后来一直在观望价钱 ，真是越观望越不敢买，我以为去年到底了，结果今年更低了

还会更低的，静观其变

**Reply by qq912276337** [reply]
哪有那么多刚需啊？？？可买可不买就不叫刚需

**Reply by money4marry** [reply]
你买的是钢筋混凝土使用权，不是土地，不要当成买不动产了，就是一次消费。
如果房价持续上涨，可以短期获利，得有更多的人冲进来买房才有可能实现，我对这个看衰
如果房价下跌，你以为是传家宝，再过20年看看是危房不（预售房看不出物业好坏吧，维护不当的楼房住起来什么感受）

我买了房，30年贷款，很焦虑。期房

**Reply by WallStreetHu** [reply]
> 19年想买 ，已经在看了， 后来一直在观望价钱 ，真是越观望越不敢买，我以为去年到底了，结果今年更低了

房价对比下就知道，国内房价除了一线，全部虚高，房价收入比可能世界第一，还没到发达城市，你这破房子比发达国家还贵，武汉这地方房子比达拉斯、圣安东尼奥还贵，你说正常吗，人家是别墅，你这边小区房只能称为公寓。

**Comment by qq912276337** [comment]
分化一部分，安抚一部分，解决一部分

**Comment by sibs-wang** [comment]
但是你要交社保养老做核酸啊
